### PR TITLE
Python 3 - pathod.language.actions

### PR DESCRIPTION
**Pull request by dufferzafar**

### CODE REVIEW

File: pathod/language/actions.py [hunk 1]
```diff
@@ -31,8 +31,11 @@ def resolve(self, settings, msg):
             c.offset = l + 1
         return c
 
-    def __cmp__(self, other):
-        return cmp(self.offset, other.offset)
+    def __lt__(self, other):
+        return self.offset < other.offset
+
+    def __eq__(self, other):
+        return self.offset == other.offset
```
Comment (mhils) [inline pathod/language/actions.py]: Add `@functools.total_ordering` here?

**Comment by Kriechi** [comment]
@dufferzafar seems a few tests are still failing:

For example: `test/pathod/test_language_actions.py:65`
The test calls: `language.parse_pathod("400:i0,'foo'")`
but `parse_pathod` calls `decode("ascii")` on the input string. So we need to change the data type here again. 
I think the other test failures are related to this.

**Comment by dufferzafar** [comment]
Sorry, but I failed to mention that this pull should be merged AFTER #1170 gets merged as I have fixed that particular issue [here](https://github.com/mitmproxy/mitmproxy/pull/1170/files#diff-62a6d1306d5663b250cf12a5a73ce0e1R34).

**Comment by dufferzafar** [comment]
Stalled because one of the tests is failing, and it depends on proper porting of `pathod.langauge.base`

**Comment by dufferzafar** [comment]
Waiting for #1210

**Comment by dufferzafar** [comment]
I think this pull can be merged as nothing more needs to be done here. A test fails currently, but it'll pass once #1210 gets merged (I've ran the tests with the branch merged locally.)

**Comment by Kriechi** [comment]
Could you please rebase to resolve the conflicts?

**Comment by dufferzafar** [comment]
Done!

**Comment by dufferzafar** [comment]
I've removed Travis / Tox stuff!

**Pull request merged**